### DISCUSSION TITLE
如何正确更新铁蛋内部的jetpack版本

### DISCUSSION THREAD

**Discussion by Oldpan** [MiRoboticsLab/cyberdog_ros2#83] [Q&A] [answered]
目前铁蛋的jetpack版本是4.5.1，对应着L4T是32.5.1.
但是目前最新的jetpack版本是4.6.1，对应着L4T是32.6.1
这个更新还是蛮大的，从tensorrt7到了trt8，cuda和cudnn也都有升级。

自己也尝试直接在铁蛋内部使用jetpack官方的OTA升级方式升级，更新源、apt-get update和apt-get upgrade-dis等，不过升级完jtop和cuda驱动找不到了，cuda代码能编译但是运行不了呃。不过铁蛋的ros2是能正常跑

怀疑可能是这个ubuntu已经被修改过一些东西，在更新的时候我发现有一些warning疑似和kernel有关，或许我更新的姿势不对，，海果大大们能给点建议么。

或者我可以直接刷jetson最新版（会抹掉原来的系统），然后再搭建铁蛋环境么？
感谢！

**Comment by homalozoa** [comment]
明天我尝试一下，然后给你回复

**Comment by Oldpan** [comment]
@homalozoa 哈哈，期待啊

**Comment by homalozoa** [comment]
今天我测试了一下，先用apt mark住几个我们修改过的包，然后安装nvidia-jetpack，然后就更新完了。不过我用的那条狗貌似不太正常，在我更新之后电源板烧了……明天我再重新测试一遍，成功之后我发个帖子在Discussion里。 @Oldpan

**Comment by homalozoa** [comment]
我先把这个issue转到discussion的Q&A里

**Reply by Oldpan** [reply]
好的，期待海总的更新，不过要小心点电源板啊。、。。

**Reply by homalozoa** [reply]
不知道咋烧的。。还没查到原因哈哈哈

**Comment by 692446** [comment]
刷机不至于吧(๑•́ωก̀๑)



---原始邮件---
发件人: "Homalozoa ***@***.***&gt;
发送时间: 2021年11月25日(周四) 晚上6:58
收件人: ***@***.***&gt;;
抄送: ***@***.***&gt;;
主题: Re: [MiRoboticsLab/cyberdog_ros2] 如果正确更新铁蛋内部的jetpack版本 (Discussion #83)




 
不知道咋烧的。。还没查到原因哈哈哈
 
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by homalozoa** [reply]
对，就很迷。。

**Comment by homalozoa** [comment, marked as answer]
参见 ： https://github.com/MiRoboticsLab/cyberdog_ros2/discussions/110

**Reply by Roger200** [reply]
請問mi 用戶密碼是什么呢？还有你知道super user 的ID 和￼￼￼密碼吗？￼￼
